webpack.common.js

```diff
@@ -30,6 +30,7 @@ module.exports = (cssFilename, outputFilename, minimizeCss) => ({
     contributionsLandingPageStyles: 'pages/contributions-landing/contributionsLanding.scss',
     newContributionsLandingPage: 'pages/contributions-landing/newContributionsLanding.jsx',
     newContributionsLandingPageStyles: 'pages/contributions-landing/newContributionsLanding.scss',
+    newThankYouLandingPage: 'pages/thank-you-landing/newThankYouLanding.jsx',
```

Comment:
I'm not sure we should call this the 'newThankYouLandingPage', as it isn't really a Landing Page. Could we go with `newThankYouPage`?

Reply:
Sure thing 👍